File: POOJAVA/src/clasesPOO/UserAccesos.java
package clasesPOO;

public class UserAccesos 
{
    String idAcceso;
    String idUser;
    String idOpcion;
    String idSubmenu;
    String idMenu;
    String estado;

    public UserAccesos(String idAcc,String idUsr)
    {
        establecerDatos(idAcc,idUsr);
    }
    public String ObtenerUseracceso()
    {
       return idAcceso;
    }
    public String ObtenerUser()
    {
       return idUser;
    }
    void  establecerDatos(String idAcce,String idUs)
    {
        idAcceso=idAcce;
        idUser=idUs;
    }
}

File: POOPHP/app/Model/UserAccesos.php
<?php
namespace App\Model;
class UserAccesos
{
   public $idAcceso="";
   public $idUser="";
   public $idOpcion="";
   public $idSubmenu="";
   public $idMenu="";
   public $estado="";

   public function __construct(String $idAcc,String $idUsr)
    {
        $this->establecerDatos($idAcc,$idUsr);
    }
    public function ObtenerUseracceso()
    {
        return $this->idAcceso;
    }
    public function ObtenerUser()
    {
        return $this->idUser;
    }
    public function establecerDatos(String $idAcce,String $idUs)
    {
        $this->idAcceso=$idAcce;
        $this->idUser=$idUs;
    }
}
?>